modules/network-observability-operator-release-notes-1-10-known-issues.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+// * network_observability/network-observability-release-notes-1-10.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-operator-release-notes-1-10-known-issues_{context}"]
+= Network Observability Operator 1.10 known issues
+
+[role="_abstract"]
+Review the following known issues and their recommended workarounds (where available) that might affect your use of the Network Observability Operator 1.10 release.
+
+
+[id="eBPF-agent-compatibility-with-older-open-shift-versions_{context}"]
+== eBPF agent compatibility with older {product-title} versions
+The eBPF agent used for Packet Capture Admission (PCA) is incompatible with {product-title} (OCP) versions 4.16 and older.
+
+This limitation prevents the eBPF-based PCA from functioning correctly on those older clusters.
+
+To work around this problem, you must manually configure PCA to use an older, compatible eBPF agent container image.
+
+link:https://issues.redhat.com/browse/NETOBSERV-2358[NETOBSERV-2358]
```

Comment:
While technically true, I think this is using terms that users may not understand, and they could fear incompatibility with 4.16 or older while it's not the case.

Instead of mentioning "The eBPF agent used for Packet Capture Admission (PCA)" I would rather mention "The CLI packet capture feature", which I think is more meaningful for customers, and avoids them thinking the incompatibility could affect the operator (it does not)